Since we're using a static libgnutls.a, its dependencies must be linked in too,
or else the libcurl configure will fail to "find" it -- it tries to link a
program against libgnutls.

Normally the Libs.private should be used for this, via pkg-config --static, but
patching libcurl configure to use --static properly would be more difficult
than this hack.

--- lib/gnutls.pc.in.orig	2013-03-22 14:37:33.154455737 +0100
+++ lib/gnutls.pc.in	2013-03-22 14:37:42.082498699 +0100
@@ -18,7 +18,7 @@
 Description: Transport Security Layer implementation for the GNU system
 URL: http://www.gnu.org/software/gnutls/
 Version: @VERSION@
-Libs: -L${libdir} -lgnutls
+Libs: -L${libdir} -lgnutls -ltasn1 -lhogweed -lnettle -lgmp
 Libs.private: @LTLIBNETTLE@ @LTLIBZ@ @LTLIBINTL@ @LIBSOCKET@ @LTLIBPTHREAD@ @LTLIBICONV@ @P11_KIT_LIBS@ @LIB_SELECT@ @TSS_LIBS@ @LIB_CLOCK_GETTIME@
 @GNUTLS_REQUIRES_PRIVATE@
 Cflags: -I${includedir}
